types/crypto/index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module "crypto" {
```

Comment:
This is unnecessary now that crossEnvCrypto exists, but it helps my IDE with type definitions, so I left it